pkg/stub/stub.go

```diff
@@ -826,6 +862,10 @@ func (stub *stub) setupHandlers() error {
 		stub.handlers.RemovePodSandbox = plugin.RemovePodSandbox
 		stub.events.Set(api.Event_REMOVE_POD_SANDBOX)
 	}
+	if plugin, ok := stub.plugin.(PostUpdatePodInterface); ok {
```

Comment:
uber-nit: is there a particular reason not to put this right after UpdatePodInterface check, like you did in the other similar contexts, which does feel more intuitive as it follows the natural pod lifecycle ?

Reply:
I agree placing after UpdatePodSandbox is more intuitive. In this case I tried to follow the ordering used in this method, e.g. [the "post" methods for containers are placed at the end](https://github.com/containerd/nri/blob/v0.9.0/pkg/stub/stub.go#L849-L860), however I'm open to changing this to be more intuitive.

Reply:
Ah, then it's fine, makes more sense the way you did it. I thought we were (more globally) consistent but we're apparently not.